src/riak_ql_parser.yrl

```diff
@@ -167,8 +169,8 @@ ColumnDefinition ->
 ColumnConstraint -> not_null : '$1'.
 
 DataType -> datetime : '$1'.
-DataType -> float : '$1'.
-DataType -> int : '$1'.
+DataType -> float_type : '$1'.
```

Comment:
It wants the type of that field to be int or float (or binary...) etc, etc so it can build its guard clauses